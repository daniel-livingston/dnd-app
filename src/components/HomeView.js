import React from "react";
import Home from "./Home";
import View from "./View";

const HomeView = () => (
	<View title='Home' description='Create or join a room to play with friends.'>
		<Home />
	</View>
);

export default HomeView;
