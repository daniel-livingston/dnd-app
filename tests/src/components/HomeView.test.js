import React from "react";
import { shallow } from "enzyme";
import HomeView from "../../../src/components/HomeView";

describe("<HomeView>", () => {
	test("renders correctly", () => {
		const wrapper = shallow(<HomeView />);
		expect(wrapper).toMatchSnapshot();
	});
});
