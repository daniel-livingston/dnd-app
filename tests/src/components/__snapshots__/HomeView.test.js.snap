// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeView> renders correctly 1`] = `
<Connect(View)
  description="Create or join a room to play with friends."
  title="Home"
>
  <Connect(Home) />
</Connect(View)>
`;
